AUTHORS

```diff
@@ -1,5 +1,3 @@
 Gabriel de Quadros Ligneul <8294320+gligneul@users.noreply.github.com>
-Pedro Argento <pedro.argento@gmail.com>
 Guilherme Dantas <guidanoli@proton.me>
-Daniel Ortega Berja <daniel.ortega@dlinn.tech>
 Zehui Zheng <work996mail@gmail.com>
```

Comment:
Pedro hasn't submitted any commits to the repo, but Daniel did. Shouldn't he remain as an author?

Reply:
I removed him because he didn't submit any changes to the remaining files of the repo.